### ISSUE TITLE
Name waypath pedestrian

### ISSUE THREAD

**Issue by Hades1503** [osmandapp/OsmAnd#11028]
hello, there is a recurring problem with the display of the names of hiking trails.

Could someone modify this display to shift it off the trail so that the name is written straight?

thank you!

![17C7A706-916A-4E35-8537-4CAA6D870623](https://user-images.githubusercontent.com/79785663/109412795-56a53180-79aa-11eb-928a-83c8b60167ac.jpeg)
![750160A8-1BDD-4BA7-8A22-2D8CD926F72B](https://user-images.githubusercontent.com/79785663/109412799-586ef500-79aa-11eb-8ccb-5d924ffc0b24.jpeg)
![9F2F715E-51B6-46E4-82ED-D788DF261FA7](https://user-images.githubusercontent.com/79785663/109412802-5ad14f00-79aa-11eb-9b92-0f2dc7183415.jpeg)

### Your Environment
OsmAnd Version:3.9.3
iOS version: 14.4
Device model: iPhone 11pro

**Maps used ( offline):**   
TOPO

**Comment by vshcherb** [comment]
Yes it's a known problem unfortunately, it's hard to fix so far.
